mesa/examples/basic/virus_on_network/Readme.md

```diff
@@ -0,0 +1,61 @@
+# Virus on a Network
```

Comment:
Why are these in two places?

Reply:
not sure what you mean?

Reply:
The old one wasn’t removed, so there are now two Readme files.